ReleaseNotes.md

```diff
@@ -1,17 +1,15 @@
-* New MicrosoftPowerBIMgmt.Data module
-    - New Get-PowerBIDataset cmdlet for getting datasets.
-    - New Get-PowerBIDatasource cmdlet for getting data sources.
-
+* Overall improvements and bug fixes
+	- Fix for netstandard.dll, copying correct assembly which allows usage in .NET 4.6.1 and Azure Automation.
+	- Fix when Update-Help is called against modules, no errors are shown.
+
+* Updated MicrosoftPowerBIMgmt.Data module
+    - New Add-PowerBIDataset for adding datasets to workspaces.
+    - New Add-PowerBIRows for adding rows to a table.
+	- New Get-PowerBITable for getting tables.
```

Comment:
I'm not sure what update period these notes are supposed to cover, but if you check our documentation, you can see that some of the cmdlets you have listed here have already been released.

https://docs.microsoft.com/en-us/powershell/module/microsoftpowerbimgmt.data/index?view=powerbi-ps
https://docs.microsoft.com/en-us/powershell/module/microsoftpowerbimgmt.reports/index?view=powerbi-ps

Reply:
Wasn't advertised though so even though they were in the previous version they are supported now